.gitignore

```diff
@@ -17,4 +17,5 @@ eclcc.log
 ./helm/examples/azure/sa/env-sa
 ./dockerfiles/platform-build-incremental/hpcc.gitpatch
 .env
-vcpkg.json
\ No newline at end of file
+vcpkg.json
+rsync_include.txt
```

Comment:
This could be moved into one of the mounted volumes so wouldn't need this exclusion.